packages/frontend/src/view/components/tables/l2-transactions/perpetual/PerpetualL2TransactionsTable.tsx

```diff
@@ -20,16 +20,17 @@ import { PerpetualL2TransactionFreeForm } from './PerpetualL2TransactionFreeForm
 export interface PerpetualL2TransactionsTableProps {
   transactions: PerpetualL2TransactionEntry[]
   collateralAsset: CollateralAsset
+  showDetails: boolean
 }
 
 export function PerpetualL2TransactionsTable(
   props: PerpetualL2TransactionsTableProps
 ) {
   const columns: Column[] = [
-    { header: 'Age' },
+    { header: `ID` },
```

Comment:
Maybe we could have an eslint rule for no unnecessary backticks? Not sure if a rule like this exists